gallery/src/data-visualization/ReactVisDonutChartPage.tsx

```diff
@@ -0,0 +1,180 @@
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>colorType</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Override to let React-vis know that the colors provided are literal color values
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>innerRadius</NxTableCell>
+              <NxTableCell>number (in pixels)</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                If radius is not set on the data then this can be used to set the innerRadius for all of the rows.
+                This can be useful for building donut charts.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>labelsRadiusMultiplier</NxTableCell>
+              <NxTableCell>number</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                How far the labels should be from the center of the chart as a function of the radius of the chart.
+                If not specified, the default value of 1.1 is used (slightly outside of the chart).
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>onValueMouseOut</NxTableCell>
+              <NxTableCell>function</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                This handler is triggered either when the user's mouse leaves a mark. The handler passes two arguments,
+                the corresponding datapoint and the actual event.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>onValueMouseOver</NxTableCell>
+              <NxTableCell>function</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                This handler is triggered either when the user's mouse enters a mark. The handler passes two arguments,
+                the corresponding datapoint and the actual event.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>padAngle</NxTableCell>
+              <NxTableCell>number/function</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                The padding to be applied between arcs.
```

Comment:
If it's a function, does it take arguments?

Reply:
> If it's a function, does it take arguments?

Not entirely sure. Official docs do not provide details.
![image](https://user-images.githubusercontent.com/7110862/126369931-4288435a-afca-4dd8-b721-ccce68bad8ef.png)

Reply:
> Not entirely sure. Official docs do not provide details.

I think we should either figure it out or just not document the `function` case